packages/react/src/components/Authenticator/Authenticator.tsx

```diff
@@ -1,7 +1,14 @@
 import * as React from 'react';
-import { AuthenticatorMachineOptions } from '@aws-amplify/ui';
+import {
+  AuthenticatorMachineOptions,
+  CognitoUserAmplify,
+} from '@aws-amplify/ui';
 
-import { Provider, useAuthenticator } from './hooks/useAuthenticator';
+import {
+  Provider,
+  useAuthenticator,
+  UseAuthenticator,
```

Comment:
Should this be a type import to be extra explicit? I think we should start distinguishing our imports because it would make them easier to read.
```
import type {  UseAuthenticator } from './hooks/useAuthenticator';
```